data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -159,15 +138,28 @@
     "viewing_hint": "CV",
     "range": "???"
   },
-  "contributor": [
-    "array of strings (CV)"
-  ],
-  "related_objects": [
-    {
-      "relationship_type": "CV",
-      "object": "ARK",
-      "position": "int",
-      "layer": "int"
-    }
-  ]
+  "related_objects": {
```

Comment:
This is based on what we discussed last week where we have a predetermined set of 'top-level' relationship types ('has_part', etc.) each with their own list of available fields, including a mechanism for further specifying the nature of the relationship with `type`.